docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -164,7 +179,7 @@ Prior to testing a tax provider, the merchant or partner test store should have
 * The tax provider, found by navigating to **Settings > Setup > Tax**, is enabled
 * If document submission is supported, navigate to **Settings > Setup > Tax > _TaxProviderName_** and ensure the **Submit Order Data** checkbox is checked
 
-To test the tax provider connection, navigate to **Settings > Setup > Tax > _TaxProviderName_** in an active MSF-enabled BigCommerce control panel and click **Test Connection**. This will trigger a sample tax estimate request to be sent to the estimate URL provided by the tax provider. If the connection is unsuccessful, users can navigate to **Settings > Advanced > Store logs** in an active MSF-enabled BigCommerce control panel to view the error and its description to assist with triaging the issue.
+To test the tax provider connection, navigate to **Settings > Setup > Tax > _TaxProviderName_** in an active MSF-enabled BigCommerce control panel and click **Test Connection**. This will trigger a sample tax estimate request to be sent to the estimate URL provided by the tax provider. If the connection is unsuccessful, merchants can navigate to **Settings > Advanced > Store logs** in an active MSF-enabled BigCommerce control panel to view the error and its description to assist with triaging the issue.
```

Comment:
Minor question, should this be changed to merchants or **both** merchants and partners? (i.e. do partner stores have the same UI as merchant stores?)

Reply:
Got my answer [here](https://github.com/bigcommerce/dev-docs/pull/1728#discussion_r1152568576)